internal/api/service/service.go

```diff
@@ -56,9 +55,10 @@ func (s *Service) getReleaseStatus(_ *models.Release) release.Status_Code {
 	return release.Status_PENDING_INSTALL
 }
 
-func (s *Service) getTravisURL(ctx context.Context) string {
-	url, err := s.client.GetTravisCIBuildURLForRef(ctx, s.repository, s.ref)
+func (s *Service) getTravisURL(ctx context.Context, r models.Release) string {
```

Comment:
I think it makes sense to take a `*models.Release`, since the Release struct is much larger than a pointer, and we avoid awkwardly dereferencing the reference we just created in the for-loop of `ListReleases`